apps/web/next.config.js

```diff
@@ -94,11 +94,11 @@ const nextConfig = {
     "@calcom/ui",
   ],
   modularizeImports: {
-    "@calcom/ui/components/icon": {
-      transform: "@react-icons/all-files/fi/{{member}}",
-      skipDefaultConversion: true,
-      preventFullImport: true,
-    },
+    // "@calcom/ui/components/icon": {
+    //   transform: "@react-icons/all-files/fi/{{member}}",
+    //   skipDefaultConversion: true,
+    //   preventFullImport: true,
+    // },
```

Comment:
is this a good idea @zomars?

Reply:
@abdallahalsamman why was this removed? i think we can keep the transformer, we just need to change the import schema

Reply:
@PeerRich should be fixed :)